### ISSUE TITLE
Form redesign

### ISSUE THREAD

**Issue by fannyBkane** [emfoundation/ce100-app#1125]
- Update masthead colour for them - blue/grey? 
- Make the input fields just have a bottom-border not border all around them.

**Comment by fannyBkane** [comment]
Hey @asimonw these are the amends, to improve the forms across the app. Ordered by priority. will add screenshots shortly.

**- Font-size**
H1
H2 (profile title)
H3 (form label)
Form input

**- Font-color**

**- Margins/paddings**

**- lines between sections**

**- Input fields style/interaction**

**Comment by Danwhy** [comment]
@fannyBkane 
The root/html font size is set to **16px** and the body is set to **14px**
We are using rems for the font size hierarchies (h1, h2 etc) and all margins/padding

Here are all the colors pulled from the site:
* #fafafa
* #003253
* #ffffff
* #585667
* #f2f2f2
* #333333
* #002944
* #4f4e5d
* #fa3265
* #e6e6e6
* #00233b
* #274d65
* rgba(64, 102, 126, 0.25)
* rgba(255, 255, 255, 0.07)
* #fff4f4
* #89a0ae
* #708795
* #4bc097
* #e33838
* #e3dc38
* #0d334b
* #dfdfdf
* #9cafbb
* #8396a2
* #ffe2e2
* rgba(102, 216, 224, 0.1)
* #e4164c
* #ff5e5e
* #5db4c0
* rgba(255, 255, 255, 0.7)
* #cfd3d6
* rgba(64, 102, 126, 0.75)
* #999999
* #d8d8d8
* rgb(0, 50, 83)
* #3f3d4e
* #e1194c
* rgba(252, 92, 100, 0.1)
* #fc5c64
* #033352
* rgba(64, 101, 126, 0.1)

**Comment by sophielevens** [comment]
Hi @fannyBkane, 

Is it okay for me to begin styling the forms today with the mockups you've provided on Invision (see below)?

![image](https://user-images.githubusercontent.com/25961258/41769502-64950102-7607-11e8-917f-e0b92797aa0f.png)

Following our conversation yesterday regarding a cello style guide I wasn't sure if I should wait for this from you.

Thanks 😄

**Comment by fannyBkane** [comment]
Hi @sophielevens the amends are ordered by priority I hope it makes sense


### Font-size and weight


Body font size changes to 16px on both mobile and desktop (across the app...), which means than every piece of text on the forms will be 1rem, except:

![screen shot 2018-06-22 at 09 59 02](https://user-images.githubusercontent.com/20577339/41768058-25d207de-7603-11e8-8b53-61f4bb480664.png)
18px (1.125rem) and weight semi bold


![screen shot 2018-06-22 at 10 06 58](https://user-images.githubusercontent.com/20577339/41768401-070eb5bc-7604-11e8-8a7e-b1faa8422729.png)
14px (0.875rem) and italic


![screen shot 2018-06-22 at 09 59 06](https://user-images.githubusercontent.com/20577339/41768101-3dd2bf18-7603-11e8-9248-609a999c9ba5.png)
14px (0.875rem) and italic


### Colours

- All text, input field underline, lines separating sections, edit buttons (outline, text and icon) changes to the same grey #999999 (that grey comes from the list above. It has a low contrast but i'd like to trial it. So it's helpful if all those elements use a colour variable that can be easily changed)
- The H2 are Dark grey #333333
- The links are Lavender #585667
- When you start writing text in the input fields, the colour of the text is dark grey #333333
- Primary 'Save' buttons are pink, secondary buttons i.e. 'Choose picture' are Navy
- The masthead background fill use the gradient **on mobile**
- The masthead is white with a H2 font-colour Navy **on desktop**
- The toggles have either a Lavender or a Navy fill 
![screen shot 2018-06-22 at 10 09 10](https://user-images.githubusercontent.com/20577339/41768555-6f7015d8-7604-11e8-88ad-0aacff830447.png)

### Margins

Please see margins on Invision (links below). They should all correspond to multiple of the root size – optimised for rem use.

### Input field transition on active
Same transition as comment field on challenges

### Layout on desktop

### Layout on tablet
Follows mobile layout

Please see layout on Invision (links below) 
 
**Note:** please don't take into account the change of icon on mobile (arrow instead of cross)  

**MOCKUPS**

Mobile:
[my personal settings](https://invis.io/3DEU1V0A6#/299074281_My_org_settings)
[my org settings](https://invis.io/3DEU1V0A6#/299074282_My_personal_settings)
[new challenge](https://invis.io/3DEU1V0A6#/302376377_Add_challenge)

Desktop
[my personal settings](https://invis.io/DYESHUZW7#/302664728_My_Menu-My_Settings)

**Comment by sophielevens** [comment]
Hey @fannyBkane,

Thanks for this, I just have a couple of questions:

1.  We currently have a background colour of #fafafa throughout the application but in your designs this is #ffffff, is this going to be implemented on the forms in this sprint or is it a change that will happen throughout the app? 

2. Would you like the edit button to have this style throughout the application or just on the forms?

**Comment by sophielevens** [comment]
Hey @fannyBkane,

Thanks for the above 😄 

On the user edit profile designs the email field has been removed completely, is this correct and a change that I should implement?

**Comment by asimonw** [comment]
@fannyBkane @sophielevens We will not include this one in this sprint. We will break this and related matters up into several issues to better clarify priorities in a next sprint!

**Comment by asimonw** [comment]
@fannyBkane Maybe some of this can now be considered in the context of #1489 ?

**Comment by fannyBkane** [comment]
That's right @asimonw - working on it this week so will move relevant items across to #1489
